src/main/resources/email-templates/pending-register-checks.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Pending register checks</title>
+</head>
+<body>
+    <p>A total of ${totalStuck} register checks have been pending for more than ${expectedMaximumPendingPeriod}.</p>
+    <br>
+    <table>
+        <thead>
+        <tr>
+            <th>GSS code</th>
+            <th>Register check count</th>
+        </tr>
+        </thead>
+        <tbody>
+            ${pendingRegisterCheckResultsHtml}
+        </tbody>
+    </table>

```

Comment:
Having this be an unstyled HTML table might be too weird - I'll consider making these <p> tags